site/src/pages/api/auth/roadmap.ts

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
```

Comment:
You have out this in the `auth` folder.  Move it up 1 level so that the url you hit is `/api/roadmap`